frontend/packages/shipwright-plugin/src/components/build-tabsection/TriggerLastBuildButton.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { Button } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import { impersonateStateToProps } from '@console/dynamic-plugin-sdk';
+import { errorModal } from '@console/internal/components/modals';
+import { useAccessReview } from '@console/internal/components/utils';
```

Comment:
`useAccessReview` from the internal package is deprecated, please use `useAccessReview` from the dynamic-plugin-sdk:

```suggestion
import { useAccessReview } from '@console/console-dynamic-plugin-sdk';
```